packages/core/src/repository/query/network-query.ts

```diff
@@ -0,0 +1,20 @@
+import { ParameterType } from '../../data';
+
+export abstract class NetworkQuery {
+    constructor(public readonly endpoint: string) {}
+
+    get body(): string | FormData {
```

Comment:
See comment regarding `AngularHttpRequestBuilder.body`; we could encode "no body" as `undefined` instead of using an empty string?